packages/core/src/utils/getMultipleChallengeConfirmData.ts

```diff
@@ -18,7 +18,7 @@ export const getMultipleChallengeConfirmData = async (assertionIds: number[]): P
     const refereeCalculationsContract = new ethers.Contract(config.refereeCalculationsAddress, RefereeCalculationsAbi, provider);
     
     // Call the contract to retrieve confirmation data and hash for the provided assertion IDs
-    const [confirmData, confirmHash]: [string[], string] = await refereeCalculationsContract.getConfirmDataMultipleAssertions(assertionIds);
+    const [confirmData, confirmHash]: [string[], string] = await refereeCalculationsContract.getConfirmDataMultipleAssertions(assertionIds, config.rollupAddress);
```

Comment:
@XDapps Please have a look at this, how come this was not needed when you tested for the challenges ?

Reply:
@CryptITAustria , the testing requested in [this ticket here](https://www.pivotaltracker.com/story/show/187985818) has never been completed because it was blocked by this story/PR that we are commenting on now. 

We cannot complete the testing until this PR is approved and merged.

Reply:
There may be more testing to be done for that ticket, but this function was tested successfully on this ticket 😄